runtime/development/src/lib.rs

```diff
@@ -123,7 +123,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("centrifuge-devel"),
 	impl_name: create_runtime_str!("centrifuge-devel"),
 	authoring_version: 1,
-	spec_version: 1011,
+	spec_version: 1015,
```

Comment:
When do we increase this one and why so many steps now?

Reply:
I need to increase this everytime I run a runtime upgrade on our moonbase dev parachain. The reason why it's 4 deploys so far it's because things didn't work right away :p for example when the XCM Barrier didn't allow for Transact messages starting with a DescendOrigin

Reply:
Yeah for sure. On mobile development usually apps have a "build_number" and a version, which allows devs to deploy new (development) versions with a bumped build number without impacting the "public" release version.